src/UberStrok.WebServices/ServerManager.cs

```diff
@@ -1,7 +1,8 @@
-﻿using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Linq;

```

Comment:
Is this `using` actually being used?

Reply:
Yes, on this line here
`var servers = Utils.DeserializeJsonAt<JObject>("configs/game/servers.json");`
In <JObject> as Newtonsoft.Json.Linq.Jobject

Reply:
Ahh I see.